.github/config/user-data

```diff
@@ -0,0 +1,32 @@
+#cloud-config
+# Set the default user
+system_info:
+  default_user:
+    name: netty
+
+# Unlock the default user
+chpasswd:
+  list: |
+     netty:netty
+  expire: False
+
+# Other settings
+resize_rootfs: True
+ssh_pwauth: True
+timezone: Europe/Berlin
+
+packages:
+ - java-11-openjdk-devel
```

Comment:
it would be nice test it on different java versions? or what do you think?

Reply:
I would prefer to keep it with one version for now as its sloooooow